client/state/comments/reducer.js

```diff
@@ -319,6 +321,31 @@ export const treesInitialized = keyedReducer(
 	keyedReducer( 'status', treesInitializedReducer )
 );
 
+/***
+ * Stores the active reply comment for a given siteId and postId
+ * @param {Object} state redux state
+ * @param {Object} action redux action
+ * @returns {Object} new redux state
+ */
+export const activeReplies = createReducer(
+	{},
+	{
+		[ COMMENTS_SET_ACTIVE_REPLY ]: ( state, action ) => {
+			const { siteId, postId, commentId } = action.payload;
+			const stateKey = getStateKey( siteId, postId );
+
+			// If commentId is null, remove the key from the state map entirely
+			if ( commentId === null ) {
+				return omitBy( state, ( value, key ) => {
```

Comment:
i'd use `omit` instead of `omitBy`.  omit is quicker and cleaner in the case where you have the key.
```js
return omit( state, stateKey );
```

as opposed to: 
```js
return omitBy( state, (value, key) => key === stateKey );
```

Reply:
Thanks @samouri! Much simpler.